cmd/operator-sdk/new/cmd.go

```diff
@@ -389,6 +394,8 @@ func verifyFlags() error {
 	return nil
 }
 
+// Deprecated: the git-init flag was deprecated since has no need to make the command run the git init.
```

Comment:
you could in addition to adding the deprecation move it to another file, so when the time comes you just need to delete the file and make the code build again

Reply:
I do not think that it is required.